store/src/java/com/zimbra/cs/mailbox/util/TypedIdList.java

```diff
@@ -224,6 +224,16 @@ public boolean contains(Integer id) {
         return ids;
     }
 
+    public int getMaxModSequence() {
+        int mod = 0 ;
+        for (List<ItemInfo> set : type2ids.values()) {
+            for (ItemInfo itemInfo : set) {
+                mod = itemInfo.getModSequence();
```

Comment:
Doesn't this need to only set it if it the new value is larger than the current one?

Reply:
Database query is order by sequence number.

Reply:
Perhaps a comment to that effect to avoid future people worrying about it?